maintainers/maintainer-list.nix

```diff
@@ -6589,6 +6589,11 @@
       fingerprint = "78C2 E81C 828A 420B 269A  EBC1 49FA 39F8 A7F7 35F9";
     }];
   };
+  hulr = {
```

Comment:
Please consider adding another mean of contact

Reply:
Thanks for your feedback. I tried to incorporate it but would rather not like to add another mean of contact.